docs/modules/ROOT/pages/index.adoc

```diff
@@ -251,6 +254,68 @@ NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issue
 [#describe-actor-role]
 NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/26[issue #26]): Describe credential holder’s role in relation to the asset.
 
+
+=== Security Considerations for Creating Identity Assertions
+
+Identity Assertions primarily allow a _<<_credential_holder,credential holder>>_ to 
+form a non-repudiable, tamper-evident binding between the _<<_credential_subject,credential subject>>_ and the list of _<<_referenced_assertions,referenced assertions>>._  In some cases, the identity assertion provides an additional trust signal over the information provided by the C2PA Claim Signature, and in other cases, the Identity Assertion is of primary interest, and the Claim Signature provides supplemental information.  
+
+The Identity Assertion straightforwardly supports the former (trust-signal) case, 
+because the Claim Signature integrity protects the identity assertion from stripping or replacement.  This means that the identity assertion cannot be tampered without invalidating the Claim Signature.
+
+However, the converse is not true. Without additional protections, the identity assertion 
+does not guard against the Claim Signature being stripped and replaced.  The mechanisms
+described in this section provide protections against this attack.
+
+The Claim Signature protects against stripping or modification of an identity assertion 
+because the hash of the identity assertion is included in the Claim Signature.  Unfortunately, it is not possible for the identity assertion to include the hash of the 
+Claim Signature because it is not known when the identity assertion is created.  This 
+specification provides a weaker binding that allows the identity assertion creator to 
+specify the key/certificate that is authorized to sign the Claim. This does not prevent a 
+Claim Signature being stripped and replaced by the _same_ Claim Signer, but it does prevent
+the Claim Signature from being replaced by a different Claim Signer.  To enable 
+this protection, the identity assertion creator can optionally include the hash of
+the Claim Signer credential/certificate authorized to (subsequently) sign the Claim.  If these
+protections are not needed (for example, if the identity assertion is expected to be
+a supplemental trust signal), then the `claim_certificate_hash` field can be omitted.
```

Comment:
I think that a soft-binding is a minimum requirement and hard-binding to the image needs to be supported. It only takes one tiny mistake for an attacker to be able to replay someone's identity forever.
The problem with optional security fields like this is that they are based on the premise that a very large ecosystem of individual organizations will all be incredibly intelligent, carefully read every line of two extremely complicated and lengthy specs, and flawlessly come to the right conclusion regarding a critical security decision every single time. In reality, the way that these types of flags almost always play out is that enough people get it wrong to the point where it becomes intolerable for the ecosystem, and then the system makes it mandatory. For instance, everyone should obviously be smart enough to not check secrets into public GitHub repositories. However, GitHub offers built-in secret detection for a reason. Missing this flag would be just as easy, if not easier.